visidata/loaders/hdf5.py

```diff
@@ -42,10 +42,16 @@ def iterload(self):
 
 
     def openRow(self, row):
+        import h5py
         if isinstance(row, BaseSheet):
             return row
-        if isinstance(row, h5py.Object):
-            return H5ObjSheet(row)
+        if isinstance(row, h5py.HLObject):
+            return Hdf5ObjSheet(row)
+
+        import numpy
+        from .npy import NpySheet
```

Comment:
Is the `from .npy` needed? `from visidata import *` fails to import it?

Reply:
While I recognized `import *` is used all over the place throughout the code base, it is considered bad practice in general (no offense!). So, my mind didn't actually lean towards not having to import specific things ;].

However, I just tried to remove that line. But then, VisiData croaks:

![image](https://user-images.githubusercontent.com/453543/96199029-ba2c4680-0f56-11eb-8066-748f85c349d7.png)

Reply:
This makes sense!